### PR TITLE
followup edits to PR3823

### PR DESCRIPTION
**Pull request by bfallonf**
As per: #3823 

Putting the Ipsec topic into the docs, and some followups.

@ahardin-rh @bmcelvee @adellape peer review plz?

### CODE REVIEW

File: admin_guide/ipsec.adoc [hunk 1]
```diff
@@ -38,10 +38,13 @@ with a host that does.
...
-Ensure that the *libreswan* package, version 3.15 or later, is installed
-on cluster hosts. If opportunistic group functionality is required then
-*libreswan* version 3.19 or later is required. At this time *libreswan*
-3.15 is the latest version supported on Red Hat Enterprise Linux 7.
+
+At this time *libreswan* version 3.15 is the latest version supported on Red Hat
+Enterprise Linux 7. Ensure that the *libreswan* 3.15 or later is installed on
+cluster hosts. If
+xref:../admin_guide/ipsec.adoc#admin-guide-ipsec-opportunistic-group-configuration[opportunistic
+group functionality] is required then *libreswan* version 3.19 or later is
```
Comment (bmcelvee) [inline admin_guide/ipsec.adoc]: Tiny thing, add a comma before 'then'.

**Comment by ahardin-rh** [comment]
No further edits from me :trophy:

**Comment by bfallonf** [comment]
Thanks @bmcelvee @ahardin-rh . Updated. I'll merge.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../admin_guide/ipsec.adoc#admin-guide-ipsec[Encrypting Hosts with IPsec]
|Re-added the Encrypting Hosts with IPsec topic and made it specific to the current version of *libreswan*.
%

**Pull request merged**